clm/config/config_gpu.yaml

```diff
@@ -0,0 +1,35 @@
+model:
```

Comment:
Thank you for adding the `config_gpu.yaml`.

Do you think we can align the same [config.yaml](https://github.com/ZhiyuLi-goog/MoE_study/blob/main/clm/config/config.yaml#L1-L4)?

I think that will help ablation comparison between CUDA and XLA in the long run which just ensure the exact same exp configuration.